README.md

```diff
@@ -1,22 +1,27 @@
-[![Build Status](https://samhstn.com/badge)](https://console.aws.amazon.com/codesuite/codebuild/projects/Push)
+# [samhstn.com](http://samhstn.com)
```

Comment:
Are you still configuring the DNS for https://samhstn.com ? 💭 
![image](https://user-images.githubusercontent.com/194400/75193456-3666e500-574e-11ea-9f9c-2497afc217ae.png)

Hope your evening is going well. 😉

Reply:
Hey Nelson! How are you? I'd love to have a catch up soon if you're game!

Yes, it's still in progress 😬I'm looking to prioritize this over the coming weeks though

Feel free to unwatch if these notifications become too frequent btw - there may be quite a few incoming.

I can assign when ready for review if you'd like - let me know what works for you!

Reply:
"_unwatch_" ...? _never_! I learn so much from watching your work! 😍
always up for catching up! looking forward to seeing your site when you re-launch it! 👍